src/applications/vaos/containers/ExpressCareFormPage.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { FETCH_STATUS } from '../utils/constants';
+import FormButtons from '../components/FormButtons';
+
+import * as actions from '../actions/expressCare';
+
+function ExpressCareFormPage({
+  submitStatus,
+  submitExpressCareRequest,
+  router,
+  routeToPreviousAppointmentPage,
+}) {
+  return (
+    <div>
+      <h1>Express Care form</h1>
+      <FormButtons
+        backBeforeText=""
+        backButtonText="Back"
+        nextButtonText="Submit Express Care request"
+        pageChangeInProgress={submitStatus === FETCH_STATUS.loading}
+        disabled={submitStatus === FETCH_STATUS.failed}
+        loadingText="Submitting your Express Care request"
+        onBack={() => routeToPreviousAppointmentPage(router, 'form')}
+        onSubmit={() => submitExpressCareRequest(router)}
+      />
+      {submitStatus === FETCH_STATUS.failed && (
```

Comment:
I will fill this error handling out more in a separate PR, because this one is already too big.